src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/specifications/PersonSpecification.kt

```diff
@@ -55,25 +55,24 @@ object PersonSpecification {
           )
         }.toTypedArray()
         criteriaBuilder.or(*postcodePredicates)
-      }
+      } ?: criteriaBuilder.disjunction()
```

Comment:
❤️ I was reading an article yesterday when I learned what this method does (I think). It was in the Friday list in #hmpps_dev